test/unit/util.test.ts

```diff
@@ -30,3 +31,27 @@ test('util -> Throttle', async (t) => {
     t.equals(sanitizeString('SELECT * FROM db WHERE something="Poler"'), 'SELECT   FROM db WHERE something  Poler ', 'Escaping query characters');
 
 });
+
+test('Encoding ArticleId for Zim html Url', async(t) => {
```

Comment:
The way the test is build is not really nice because:
* It repeat a lot of time the same piece of code
* It put the input value, output value and comment in difference places

A better way to do so is creating an array with all the values an iterate through it.

Reply:
The right way to write HTML is HTML (https://en.wikipedia.org/wiki/HTML) not `html`. It is a free string, we are free to write it properly.